bundles/core/src/main/java/com/adobe/cq/commerce/core/search/internal/services/SearchFilterServiceImpl.java

```diff
@@ -30,16 +30,7 @@
 import com.adobe.cq.commerce.core.search.models.FilterAttributeMetadata;
 import com.adobe.cq.commerce.core.search.services.SearchFilterService;
 import com.adobe.cq.commerce.graphql.client.GraphqlResponse;
-import com.adobe.cq.commerce.magento.graphql.Attribute;
-import com.adobe.cq.commerce.magento.graphql.AttributeInput;
-import com.adobe.cq.commerce.magento.graphql.CustomAttributeMetadata;
-import com.adobe.cq.commerce.magento.graphql.CustomAttributeMetadataQueryDefinition;
-import com.adobe.cq.commerce.magento.graphql.Operations;
-import com.adobe.cq.commerce.magento.graphql.Query;
-import com.adobe.cq.commerce.magento.graphql.QueryQuery;
-import com.adobe.cq.commerce.magento.graphql.__InputValue;
-import com.adobe.cq.commerce.magento.graphql.__Type;
-import com.adobe.cq.commerce.magento.graphql.__TypeQueryDefinition;
+import com.adobe.cq.commerce.magento.graphql.*;
```

Comment:
hmmm ... no `*` import please (you can configure that in your IDE)

Reply:
I think I fixed it for good this time